backend/src/main/resources/db/migration/V25__Alter_match_table.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE matches DROP COLUMN child1_id;
+ALTER TABLE matches DROP COLUMN child2_id;
```

Comment:
Why did you drop the child columns? Don't you still need them?

Reply:
Wait now I see a match_child1 and match_child2. What is going on?

Reply:
We have `team1_id`, `team2_id`, `match_child1`, `match_child2`. At two of them will be not null at any given time. For first round matches, it will be `team1_id` and `team2_id`. Other matches will have `match_child1` and `match_child2` start out as not null. Once a match is finished it's parent is updated such that teamX_id is filled in.